test/db.js

```diff
@@ -37,49 +35,54 @@ const testCases = {
 		name: 'Kaarel Kehtetu',
 		examiner: 'Tiit Testija',
 		examTime: '2017-11-15',
-		formFillTime: null,
 		expiryTime: '2021-03-06'
 	},
-	'40000000000': {
-		id: '40000000000',
-		certificate: 'none',
-		name: '',
-		examiner: null,
-		examTime: null,
-		formFillTime: null,
-		expiryTime: null
-	},
+	// '40000000000': {
+	// 	id: '40000000000',
+	// 	certificate: 'none',
+	// 	name: '',
+	// 	examiner: null,
+	// 	examTime: null,
+	// 	expiryTime: null
+	// },
 	'50000000000': {
 		id: '50000000000',
 		certificate: 'green',
 		name: 'Agnes Aegumas',
 		examiner: 'Andrei Popov',
 		examTime: '2012-12-01',
-		formFillTime: null,
 		expiryTime: '2022-10-11'
 	},
-	'70000000000': {
-		id: '70000000000',
-		certificate: 'green',
-		name: 'Ain Vormiga',
-		examiner: 'Eerik Eksamineerija',
-		examTime: null,
-		formFillTime: '2024-01-27',
-		expiryTime: '2024-03-09'
-	},
+	// '70000000000': {
+	// 	id: '70000000000',
+	// 	certificate: 'green',
+	// 	name: 'Ain Vormiga',
+	// 	examiner: 'Eerik Eksamineerija',
+	// 	examTime: null,
+	// 	expiryTime: '2024-03-09'
+	// },
```

Comment:
If we don't use those comments, we shouldn't keep them here. Git helps us to know the history.

It looks like this example has been deleted, and another one `90000..` was created for the same purpose.

Reply:
Actually 4xx and 7xx were changed to invalidCases, see below. I deleted the comments.